consensus/src/units/validator.rs

```diff
@@ -132,63 +132,15 @@ mod tests {
     use super::{ValidationError::*, Validator as GenericValidator};
     use crate::{
         creation::Creator as GenericCreator,
-        units::{
-            FullUnit as GenericFullUnit, PreUnit as GenericPreUnit,
-            UncheckedSignedUnit as GenericUncheckedSignedUnit, Unit as GenericUnit,
+        units::tests::{
```

Comment:
These are already exported, so no `tests::` needed, right?

Reply:
Yes. Also I moved those methods to module `testing`